### ISSUE TITLE
Headings and identification of phrases

### ISSUE THREAD

**Issue by vkhatuntsevv** [dominiclet/obsidian-note-definitions#19]
for some reason, if i put a phrase on h2+, it's not detectable by the plugin anymore. i don't want to use separate files for different topics; i'd want to be able to break headings into groups, like

Quick dictionary.md
# Artifacts
## Gun
*shoot*
pistol
## Rod
catch fish
# Regions
### Leriokan
eastern region which consists of different islands
### Borrd'ghan
southern region where weather is always hot and war never ends
### Core of Korr
underground region which is unknown to most of population

unfortunately, this does not work this way. i presume, plugin can see different phrases only if they are h1

**Comment by dominiclet** [comment]
You are right, I've made the change to allow phrases/words to be detected only if they are h1 as it was the intended behaviour as was stated in the README. I understand that the current way of defining words is rather restrictive at the moment, this is something I plan to improve on subsequently, perhaps by giving more customisation options in the future.
